semantic-conventions/src/opentelemetry/semconv/templating/code.py

```diff
@@ -199,14 +201,16 @@ def get_data_multiple_files(
         return data
 
     @staticmethod
-    def setup_environment(env: Environment):
+    def setup_environment(env: Environment, trim_whitespace: bool):
         env.filters["to_doc_brief"] = to_doc_brief
         env.filters["to_const_name"] = to_const_name
         env.filters["merge"] = merge
         env.filters["to_camelcase"] = to_camelcase
         env.filters["to_html_links"] = to_html_links
         env.filters["regex_replace"] = regex_replace
         env.filters["render_markdown"] = render_markdown
+        setattr(env, "trim_blocks", trim_whitespace)
+        setattr(env, "lstrip_blocks", trim_whitespace)
```

Comment:
Please simplify this:

```suggestion
        env.trim_blocks = trim_whitespace
        env.lstrip_blocks = trim_whitespace
```